contracts/stake/src/distribution.rs

```diff
@@ -49,6 +35,40 @@ pub struct Distribution {
     pub bonus_per_day_bps: u64,
 }
 
+impl Distribution {
+    pub fn calculate_rewards_power(&self, env: &Env, staker: &Address, )
```

Comment:
```suggestion
    pub fn calculate_rewards_power(&self, env: &Env, staker: &Address, ) -> {//std::unimplemented!}
```

Reply:
You don't have to comment on work in progress and obviously unfinished stuff.